docs/azure_jumpstart_ag/contoso_supermarket/ci_cd/_index.md

```diff
@@ -4,4 +4,234 @@ weight: 100
+  - Main branch targets the local _Dev_ environment/cluster
+  - Staging branch targets the _Staging_ environment/cluster
+  - Canary branch targets the _Chicago_ environment/cluster
+  - Production branch targets the _Seattle_ environment/cluster
+
+    ![Screenshot showing the GitHub repository branches](./img/repo_branches.png)
+
+- The repository has two main folders to seperate the development team applications' code (developer) and the DevOps team operations GitOps configurations (operations)
+
+    ![Screenshot showing the GitHub repository main folder structure](./img/repo_folder_structure.png)
+
+- Within the _developer_ folder, there is a folder for each application's source code. This is where Contoso Supermarket's developers develop new features
+
+    ![Screenshot showing the developer folder structure](./img/repo_developer_structure.png)
+
+- Within the _operations_ folder, there is also folder for each application's GitOps configuration. This is where Contoso Supermarket's DevOps team manage how the applications are deployed to the different environments
+
+    ![Screenshot showing the operations folder structure](./img/repo_operations_structure.png)
+
+- Each application has folder for _charts_ where the Kubernetes mainfests for each application is located and a folder for _releases_ where the helm releases for each application and each environment is located. This way the DevOps team can control the promotion of each version of the applications across the CI/CD workflow on different environments and also enable/disable features created by the developers as needed
+
+    ![Screenshot showing the helm folder structure](./img/repo_operations_helm_structure.png)
+
+    ![Screenshot showing the helm releases folder structure](./img/repo_operations_helmreleases_structure.png)
+
+- Helm releases control the promotion of the Pos application images from one environment to the other using the image tag as a Helm value
+
+    ![Screenshot showing the helm release values](./img/github_helm_release_example.png)
+
+- The repository has the following GitHub actions that build the Contoso Supermarket's CI/CD workflow
+  - **Promte-to-staging:** This workflow is triggered once a pull request targeting the code of the Pos application is merged. It will build the new images based on code changes, increment the image tag, run unit tests and update the Pos application on the _Staging_ environment
+  - **Promte-to-canary:** This workflow is triggered once the staging cluster has a healthy deployment of the Pos application via a Flux V2 alert. It will build the new canary image, run integration tests and submits a PR to update the image tag on the _canary_ Helm release
+  - **Promte-to-production:** This workflow is triggered once the canary cluster has a healthy deployment of the Pos application via a Flux V2 alert. It will build the new production image, run final tests and submits a PR to update the image tag on the _production_ Helm release
```

Comment:
```suggestion
  - **Promte-to-production:** This workflow is triggered once the canary cluster has a healthy deployment of the Pos application via a Flux V2 alert. It will build the new production image, run final tests and submits a PR to update the image tag on the _production_ Helm release.
```